src/main/java/frc/robot/maps/RobotMap.java

```diff
@@ -1,9 +1,93 @@
 package frc.robot.maps;
 
+import com.chopshop166.chopshoplib.drive.MockSwerveModule;
+import com.chopshop166.chopshoplib.drive.SwerveModule;
 import com.chopshop166.chopshoplib.maps.RobotMapFor;
+import com.chopshop166.chopshoplib.sensors.MockGyro;
+
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj.interfaces.Gyro;
 
 // Need to get MAC address for roborio
 @RobotMapFor("Default")
 public class RobotMap {
+    public static class DriveMap {
```

Comment:
We should pull this into chopshoplib at the end of the season alongside the swerve module code.

Reply:
Would you mind calling this `SwerveDriveMap`?